lib/common.sh

```diff
@@ -11,7 +11,20 @@ DAPP_LIB=${DAPP_LIB:-$BIN_DIR/contracts}
 
 export OUT_DIR=${OUT_DIR:-$PWD/out}
 
-CONFIG_FILE="$CONFIG_DIR/$CONFIG_STEP.json"
+CONFIG_FILE="${OUT_DIR}/config.json"
+# Only executes if called from the initial script
+if [[ "$CONFIG_STEP" = "$1" ]]; then
```

Comment:
Can we use `==` here to avoid ambiguity?

Reply:
I guess so. Far away from knowing the `bash` best practices. So you say we should use `==` everywhere?

Reply:
Yes, they're [equivalent](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html)

Reply:
Done. If you can, please look the whole commit. I've done some other modifications to standardize conditionals everywhere. I might have missed some place.